spec/system/reward_spec.rb

```diff
@@ -7,25 +7,51 @@
     driven_by(:rack_test)
   end
 
-  let(:employee) { create(:employee) }
-  let!(:reward) { create(:reward) }
-
-  it 'Reward index and show for employee' do
-    sign_in employee
-    visit root_path
-    click_link 'Rewards'
-    expect(page).to have_content reward.title
-    expect(page).not_to have_content reward.description
-    expect(page).to have_content reward.price
-
-    click_link 'Show'
-    expect(page).to have_content reward.title
-    expect(page).to have_content reward.description
-    expect(page).to have_content reward.price
-
-    click_link 'Back'
-    expect(page).to have_content reward.title
-    expect(page).not_to have_content reward.description
-    expect(page).to have_content reward.price
+  context 'when the employee checks the rewards' do
+    let(:employee) { create(:employee) }
+    let!(:reward) { create(:reward) }
+
+    let!(:reward1) { create(:reward) }
```

Comment:
consider using: `FactoryBot.create_list(:reward, 10)`

Reply:
I had a problem with it, but it worked and I used create_list :)